### DISCUSSION TITLE
Group Assignment #4

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#20] [Assignments]
### Due 7/30

Submit your repository for the [rough draft](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_05/group_assignment.md) here.

**Comment by luigimagdamit** [comment]
Repository: https://github.com/LuigiMagdamitUCLA/musical-lamp
Site: https://luigimagdamitucla.github.io/musical-lamp/project/

**Comment by hrfrahn** [comment]
Repo: https://github.com/hrfrahn/191A-GroupProject
Site: https://hrfrahn.github.io/191A-GroupProject/

**Comment by edmondywen** [comment]
repo: https://github.com/edmondywen/asiaam-191a-la-heat-disparities
site: https://edmondywen.github.io/asiaam-191a-la-heat-disparities/

**Comment by emilysuh** [comment]
repo: https://github.com/emilysuh/alphabetmafia
site: https://emilysuh.github.io/alphabetmafia/

**Comment by eandrusenki** [comment]
Repo: https://github.com/eandrusenki/wehearseebelieveyou
Site: https://eandrusenki.github.io/AsAmS-191A-Assignments/titleixproject/
